vmslave-config/pipeline/vmslave_config/vmslave_config.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+#list diff files
+diff_files=`git diff --dirstat=files ${GIT_PREVIOUS_COMMIT}`
+echo $diff_files | grep "vmslave-config/"
+
+# one yml for one group
+# RE_CONFIG=true config all groups, else config changed group
```

Comment:
asking from Alan: RE_CONFIG means to re-config all groups in ansible host: PR-gates, builds , release...etc . all kind of slaves.